sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1594,10 +1596,18 @@ public virtual Response<RecognizeLinkedEntitiesResultCollection> RecognizeLinked
 
         internal virtual TextDocumentInput ConvertToDocumentInput(string document, string language, int id = 0)
             => new TextDocumentInput($"{id}", document) { Language = language ?? _options.DefaultLanguage };
-
```

Comment:
This line removal was probably an accident. Don't let it block you, though.

Reply:
Added back